packages/perseus/src/validation.types.ts

```diff
@@ -45,7 +44,11 @@ export type PerseusCategorizerUserInput = {
     values: ReadonlyArray<number>;
 };
 
-export type PerseusCSProgramRubric = PerseusCSProgramWidgetOptions;
+// TODO(LEMS-2440): Can possibly be removed during 2440?
+// This is not used for grading at all. The only place it is used is to define
+// Props type in cs-program.tsx, but RenderProps already contains WidgetOptions
+// and is already included in the Props type.
+export type PerseusCSProgramRubric = Empty;
```

Comment:
Wanted something here that would not add anything to the Props type in cs-program.tsx. Pretty sure `Empty` works for that, but definitely let me know if that's not true. Kind of a stop gap measure until all the iframe related widgets are reviewed.

Reply:
(I know you know, but for others that might see this)

Several widgets are wrappers around complex, nebulous things: CS Program, iFrame, Phet. IMO it's questionable that these really should even be Perseus Widgets at all. But either way, in these instances, it's the things we wrap that determine correctness and we just pass along their "score."

`Empty` works for me. I'd also be fine with `null` or something, but I don't care because we're going to circle back to this in LEMS-2440.